src/Symfony/Component/Config/Resource/ClassExistenceResource.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Resource;
+
+/**
+ * ClassExistenceResource represents a class availability.
```

Comment:
Let's replace `availability` with `existence`, to be consistent with the class name and the `class_exists` function name.